commcare-cloud-bootstrap/commcare_cloud_bootstrap.py

```diff
@@ -53,6 +53,16 @@ class AwsConfig(StrictJsonObject):
     key_name = jsonobject.StringProperty()
     security_group_id = jsonobject.StringProperty()
     subnet = jsonobject.StringProperty()
+    data_volume = jsonobject.DictProperty(exclude_if_none=True)
+    boot_volume = jsonobject.DictProperty(exclude_if_none=True)
+
+    @classmethod
+    def wrap(cls, data):
+        if 'data_volume' in data:
+            assert data['data_volume']['DeviceName'] == '/dev/xvdb'
+        if 'boot_volume' in data:
+            assert data['boot_volume']['DeviceName'] == '/dev/sda1'
```

Comment:
Is it our convention to use `/dev/xvdb` and `/dev/sda1`, or do AWS machines always use those devices for data and boot (that would be surprising to me)? If it's a convention, is it documented somewhere?

Unconditionally overwriting these values here seems like it could result in unexpected outcomes if `AwsConfig` is ever used in a context where other devices are being used.

Reply:
This code really has no current calling/purpose besides being used for the provable deploy, and for setting up test monoliths like this one. For that reason, I'm less concerned with things being general than I am with things working for a few predefined use-cases.

For aws EC2 instances treat `/dev/sda1` as the boot volume, so this assert I don't expect to change. (The reason I have it there at all is because in the spec file, the YAML for each disk is exactly the JSON you'd send via the `aws` command line tool, so I thought I'd just make it keep that form and check that the device is set correctly.)

For the `data_volume`, it's true this could be anything. I'd only tested it on `/dev/xvdb`, and didn't see a use for setting it to something else, so I thought I'd just have it tell you to set it to that if you had it set to anything else (or left unset), possibly avoiding an issue later with untested configuration. For now I'll just leave comments for each of these asserts with context, saying that if you ever want to, you could probably relax the `data_volume` one.

Reply:
Actually, I'm going to just remove the `/dev/xvdb` assert. I think it's more confusing than helpful.

Reply:
Wow, I don't know how I overlooked what's really going on here. My comment must have been confusing. I wrote

> Unconditionally overwriting these values...

Clearly these are asserts, and not assignments, and that makes my comment completely invalid. Sorry!